pykube/objects.py

```diff
@@ -292,6 +300,20 @@ def cordon(self):
     def uncordon(self):
         self.unschedulable = False
 
+    def drain(self):
+        node_ip = self.ip
+        namespaces = Namespace.objects(self.api)
+        for namespace in namespaces:
+            pods = Pod.objects(self.api, namespace=namespace.name).all()
+            for pod in pods:
+                annotations = pod.obj["metadata"]["annotations"]
+                if annotations and annotations.get("kubernetes.io/created-by"):
```

Comment:
Its the only place i could find out if a pod belonged to a daemonset. If i missed it elsewhere, please let me know